src/test/java/Search.java

```diff
@@ -0,0 +1,105 @@
+import InternSearch.pages.LoginPageSearch;
+import InternSearch.pages.SearchPageSearch;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class Search extends BaseTest {
+    String songName = "Dark Days";
+    //use all cases- exact spelling of song name, ingnore trailing and hading white space
+    String [] list = {"Dark Days"," Dark Days ","Dark Days "};
+
+    String [] caseSenList = {"darks day", "darKs DAy"};
+    @Test(enabled = true, priority = 0)// dataProvider = "songTitles", dataProviderClass = BaseTest.class)
+    public void searchSongTitle() throws InterruptedException{
+
+//        login();
+        LoginPageSearch Login = new LoginPageSearch(driver);
+        SearchPageSearch searching = new SearchPageSearch(driver);
+        Login.login();
+
+        for (int i=0; i < list.length; i++) {
+            //System.out.println(list[i]);
+            searching.searchBox(list[i]);
+        }
+        //check for the results
+        resultFromSearch();
+    }
+    @Test(enabled = true,priority=1)
+    public void isSongSearchwithSentiveCase() {
+        LoginPageSearch Login = new LoginPageSearch(driver);
+        SearchPageSearch searching = new SearchPageSearch(driver);
+        Login.login();
+
+        for (int i=0; i < caseSenList.length; i++) {
+            searching.searchBox(caseSenList[i]);
+        }
+        songResultpage();
+        //negativeSearchResult();
+        //Assert.assertFalse(negativeSearchResult(),"should not shown");
+        Assert.assertEquals(negativeSearchResult(), false);
```

Comment:
Consider simplifying code. For example ` Assert.assertTrue(negativeSearchResult());`

Reply:
I did this one using Assert.assertFalse(negativeSearchResult());